packages/admin/cms-admin/src/cronJobs/CronJobsPage.tsx

```diff
@@ -0,0 +1,93 @@
+import { gql } from "@apollo/client";
+import { LocalErrorScopeApolloContext, messages, Stack, Toolbar, ToolbarFillSpace, ToolbarTitleItem, useTableQuery } from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import { DataGrid } from "@mui/x-data-grid";
+import * as React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { ContentScopeIndicator } from "../contentScope/ContentScopeIndicator";
+import { GQLCronJobsQuery } from "../graphql.generated";
+
+const ScopeIndicatorLabelBold = styled(Typography)`
+    && {
+        font-weight: 400;
+        padding: 0 8px 0 4px;
+        text-transform: uppercase;
+    }
+`;
+
+const ScopeIndicatorContent = styled("div")`
+    display: flex;
+    align-items: center;
+`;
+
+const DataGridContainer = styled("div")`
+    width: 100%;
+    height: calc(100vh - var(--comet-admin-master-layout-content-top-spacing));
+`;
+
+const cronJobsQuery = gql`
+    query CronJobs {
+        cronJobs {
+            id
+            name
+            schedule
+            lastScheduledAt
+        }
+    }
+`;
+
+export function CronJobsPage(): React.ReactElement {
+    const intl = useIntl();
+
+    const { tableData, loading, error } = useTableQuery<GQLCronJobsQuery, undefined>()(cronJobsQuery, {
```

Comment:
useTableQuery is for comet-admin Table and has no real benefit with DataGrid anymore. Will be marked (together with Table) as deprecated one day.

A (imho) good example is here:
https://github.com/vivid-planet/comet/blob/main/demo/admin/src/products/ProductsTable.tsx#L97